scheduler/core/components/collector/__init__.py

```diff
@@ -503,6 +519,33 @@ def night_configurations(self,
             self.get_night_events(site).time_grid[night_idx].datetime.date() - Collector._DAY
         ) for night_idx in night_indices}
 
+    def _get_group(self, observation: Observation) -> Group:
```

Comment:
We will still need to figure out what we need if an `Observation` is a member of multiple levels of scheduling `Group`s.